src/onegov/translator_directory/layout.py

```diff
@@ -97,6 +99,17 @@ def __init__(
             request: TranslatorAppRequest
         ) -> None: ...
 
+    def translator_data_outdated(self) -> bool:
+        if self.request.is_translator:
+            tz = pytz.timezone('Europe/Zurich')
+            year_ago = datetime.now(tz=tz) - timedelta(days=365)
```

Comment:
this is not fully accurate for leap years, functionality wise minor
```
import datetime
import pytz
from dateutil.relativedelta import relativedelta

datetime.datetime.now(tz=tz) - timedelta(days=365)
datetime.datetime(2023, 8, 9, 7, 31, 3, 141404, tzinfo=<DstTzInfo 'Europe/Zurich' CEST+2:00:00 DST>)
datetime.datetime.now(tz=tz) - relativedelta(years=1)
datetime.datetime(2023, 8, 8, 7, 31, 33, 475108, tzinfo=<DstTzInfo 'Europe/Zurich' CEST+2:00:00 DST>)
```

Reply:
Wouldn't be too problematic in this case, but good to know! I changed it to year now.